IdentityCore/tests/MSIDAccountCacheItemTests.m

```diff
@@ -87,8 +87,8 @@ - (void)testInitWithJSONDictionary_whenAccount_andAllFieldsSet_shouldAccountCach
                                      };
     
     NSError *error = nil;
-    MSIDAccountCacheItem *cacheItem = [[MSIDAccountCacheItem alloc] initWithJSONDictionary:jsonDictionary error:&error];
-    
+    MSIDAccountCacheItem *cacheItem = [MSIDAccountCacheItem accountWithJSONDictionary:jsonDictionary error:&error];
```

Comment:
either add a separate test for accountWithJSONDictionary, or rename this test.

Reply:
Renamed the test